input/default.toml

```diff
@@ -114,6 +114,8 @@ author  = "Harrison Nicholls, Tim Lichtenberg"
 
 # Planetary structure - physics table
 [struct]
+
+    # Enclosed mass and radius at the atmosphere-interior interface
     mass        = 1.0       # M_earth
     radius      = 1.0       # R_earth
```

Comment:
Does this also need relabeling?

Reply:
As in renaming the variables?

Reply:
Yes that's what I meant, but on second look it seems fine.